inference-engine/ie_bridges/python/wheel/CMakeLists.txt

```diff
@@ -54,7 +54,11 @@ foreach(_target ov_runtime_libraries ie_plugins _pyngraph pyopenvino)
 endforeach()
 
 set(cp_python "cp${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
-set(openvino_wheel_name "openvino-${WHEEL_VERSION}-${WHEEL_BUILD}-${cp_python}-${cp_python}-${WHEEL_PLATFORM}.whl")
+set(cp_python_m "${cp_python}")
+if ("${cp_python}" STREQUAL "cp36" OR "${cp_python}" STREQUAL "cp37")
```

Comment:
BTW, we can use `cp_python` directly w/o `"${}"`

Reply:
It's still hard coded. I believe we should obtain these values from python:
https://github.com/openvinotoolkit/openvino/pull/9098